teps/0096-pipelines-v1-release.md

```diff
@@ -0,0 +1,104 @@
+---
+status: proposed
+title: Pipelines V1 Release
+creation-date: '2021-11-29'
+last-updated: '2021-11-29'
+authors:
+- '@lbernick'
+- '@jerop'
+- '@bobcatfish'
+- '@vdemeester'
+---
+
+# TEP-0096: Pipelines V1 Release
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Background](#background)
+- [Proposal](#proposal)
+  - [V1 Stability and Deprecation Policy](#v1-stability-and-deprecation-policy)
+  - [API Definition](#api-definition)
+  - [Use Cases](#use-cases)
+- [References](#references)
+<!-- /toc -->
+
+## Summary
+
+Today, Tekton provides Alpha and Beta APIs.
+This TEP proposes a path to releasing Tekton Pipelines V1 to provide a stable API that users can rely on.
+This TEP documents the API definition and stability policy for Pipelines V1,
```

Comment:
By API definition, do we mean the "spec" of each object we ship ? I think that's could be intersting indeed 👼🏼 
Or are we only talking about "the policy" ?

Reply:
I think what we consider the Pipelines API will be decided in a later update to this TEP, but our current answer to this question is [here](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#what-is-the-api). I am referring here to the objects we ship, but not specifically to their "spec" fields.

Reply:
Yup, I read it as "spec" of CRDs as well. You might want to replace it to: This TEP defines Tekton Pipelines V1 API and its stability/compatibility policies ... its just a nit.